onecodex/cli.py

```diff
@@ -264,7 +269,7 @@ def assets_upload(ctx, max_threads, file):
     run_via_threadpool(
         ctx.obj["API"].Assets.upload,
         [file],
-        {"progressbar": bar},
+        {"progressbar": bar, "name": name},
```

Comment:
should we validate that the name is not empty?

Reply:
The `name` option is required so it won't work if nothing is passed, but passing an empty string is permitted so I can add a guard for that case